editor/src/components/canvas/canvas-strategies/absolute-reparent-to-flex-strategy.spec.browser2.tsx

```diff
@@ -357,7 +354,7 @@ describe('Absolute Reparent To Flex Strategy', () => {
     await renderResult.getDispatchFollowUpActionsFinished()
 
     const dragDelta = windowPoint({
-      x: firstFlexChildEnd.x - absoluteChildCenter.x,
+      x: firstFlexChildEnd.x - absoluteChildCenter.x + 50,
```

Comment:
Yeah, it was removed in #2553 because with that PR it would result in an undesired re-parent. This PR fixes that by making the forced re-parent the lower weighted strategy (as the target doesn't provide coordinate bounds)